cupy/core/raw.pyx

```diff
@@ -30,10 +30,12 @@ cdef class RawKernel:
             `cuComplex.h` or not. If set to ``True``, any code that uses the
             functions from `cuComplex.h` will be translated to its Thrust
             counterpart. Defaults to ``False``.
+        enable_cooperative_groups (bool): Whether the CUDA source requires to
+            use cooperative groups.
```

Comment:
Taking it literally, you may think it must be `False` if you don't use `cooperative_groups`.
Although a bit redundant, how about something like this?

"Whether to enable cooperative groups in the CUDA source. If set to `True`, compile options are configured properly so that cooperative groups can be used from the CUDA source."

Reply:
Fixed.